fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java

```diff
@@ -364,6 +364,14 @@ public synchronized boolean cancel(boolean needWaitCancelComplete) throws JobExc
 
     @Override
     protected void executeCancelLogic(boolean needWaitCancelComplete) {
+        try {
+            if (mtmv == null) {
```

Comment:
add comment to explain when mtmv could be null

Reply:
Mtmv is initialized in the before method. If the task has not yet run, the before method will not be used, so mtmv will be empty, which prevents the canceled task from being added to the history list